### PR TITLE
Improve extensibility

### PR DESCRIPTION
**Pull request by T0RAT0RA**
Hi,
I made a few changes to improve the extensibility of this plugin:

- Adding `title` prop to `ScoreCardTable`:
image

- Adding ability to remove/change the `%` suffix in scores with `scoreSuffix`:
image
image

- Adding ability to specify `reviewer` ref object in json data, in addition to the string:
```json
    "owner": {
      "name": "team-seal",
      "kind": "Group"
    },
```

If that's something you'd consider, I'll update the tests and documentation.

#### :heavy_check_mark: Checklist

- [x] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [x] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### CODE REVIEW

**Comment by jvilimek** [comment]
Hi!
Thanks for the contribution! It's definitely something, that makes sense.  I would be more than happy to review & merge your changes when ready.

A few comments/questions if you don't mind:

- as for `Title`: I have no comment/question
- as for `scoreSuffix`: from your screenshot, it seems you want to display `A+` instead of `90 %` score, right? What does the JSON look like? How about having `scorePercent` (to be displayed with `%` suffix) and a new `scoreLabel` that shall be displayed instead if defined?
- as for generic reviewers: it seems you want to have the ability to specify a whole group as a reviewer, right? Is it because this group is assigned to do the review? Or it's because it's multiple members that did the review on date X.Y.?

Thanks a lot!

**Comment by T0RAT0RA** [comment]
- `scoreSuffix`: Yes, I pretty prefer your idea, having a different label, so it doesn't mess the sorting when `scorePercent` is not a number.
The JSON looks like: 
```json
"scorePercent": "A",
"scoreSuffix": "",
```
I'll update to to 
```json
"scorePercent": "90",
"scoreLabel": "A",
```

- Generic reviewer: Correct, I'd prefer to assign a whole team as reviewer, when they are the ones doing self-assessment of their systems.


I'll update the PR based on your comments, thanks.

File: plugins/score-card/sample-data/all.json [hunk 1]
```diff
@@ -315,5 +315,321 @@
         ]
       }
     ]
+  },
+  {
+    "systemEntityName": "podcast",
```
Comment (T0RAT0RA) [inline plugins/score-card/sample-data/all.json:320]: I copied `audio-playback.json` and added `scoreLabel` everywhere.
The file is used in tests as well.

File: packages/app/cypress/integration/scoreboard.ts [hunk 2]
```diff
@@ -27,7 +27,7 @@ describe('score-card', () => {
 
       cy.contains('System scores overview').should('be.visible');
       cy.checkForErrors();
-      cy.get('span:contains("1-2 of 2")').should('be.visible'); // beware, there is also a hidden <P/> element
+      cy.get('span:contains("1-3 of 3")').should('be.visible'); // beware, there is also a hidden <P/> element
```
Comment (T0RAT0RA) [inline packages/app/cypress/integration/scoreboard.ts:30]: We now have scoring for the system `podcast`.

**Review by jvilimek — COMMENTED** [review]
Sorry it took so long I was kind of busy. All looking good, thanks a lot for contribution. Only (I guess) small typo in one file?

**Review by rasovskym — COMMENTED** [review]
Everything looks fine.

**Review by jvilimek — COMMENTED** [review]
Great, thanks!

**Review by jvilimek — APPROVED** [review]

**Comment by jvilimek** [comment]
@T0RAT0RA FYI the change is now live ([ver 0.5.5](https://www.npmjs.com/package/@oriflame/backstage-plugin-score-card/v/0.5.5)), thanks for contribution!

**Comment by T0RAT0RA** [comment]
Thanks for the fast review and release!

**Pull request merged**